test/API/test_calls.py

```diff
@@ -0,0 +1,222 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath( os.path.join(os.path.dirname(__file__), '../../src/') ))
```

Comment:
You add the file to path? 

Why is this needed?

Reply:
Since we don't have __init__ files all over the place this is an alternative, we just add the src directory to the path.
I'd prefer this to putting init files everywhere but that is only my opinion.